problems/strings.js

```diff
@@ -103,15 +191,70 @@ function disemvowel() {}
  * @returns {string}
  */
 
- function disemvowelLongWords(){}
+// stringYouWishToSplitIntAnArray.slice(characterYouWishToSplit)
+// Popular Splits -
+// str.split(“”) - separates every character .
+// str.split(“ “) - splits words
+
+const disemvowelLongWords = (string) => {
+  let newSentence = "";
+  let noVowels = "";
+  let newString = string.split(" ");
+  for (let i = 0; i < newString.length - 1; i++) {
```

Comment:
`newString.length - 1` does not go through the entire array. You won't be able to reach the last element in the array